src/components/dashboard/FeaturedApps/FeaturedApps.tsx

```diff
@@ -11,10 +11,14 @@ import SafeAppIconCard from '@/components/safe-apps/SafeAppIconCard'
 
 const StyledGrid = styled(Grid)`
   gap: 24px;
+  height:100%;
 `
 
 const StyledGridItem = styled(Grid)`
   min-width: 300px;
+  @media (min-width: 1460px) {
+    min-width: 500px;
+  }
```

Comment:
We can get rid of the min-width styles if we instead change the `flex-direction` on the `StyledGrid` element like so: `<StyledGrid container flexDirection={{ xs: 'column', sm: 'row', lg: 'column' }}>`

Reply:
HI, @usame-algan, i have made the required changes.

Reply:
@usame-algan  i messed up and now its showing unwanted changes in files changed. What should i do?

Reply:
Looks good! You could git reset back to the first commit on this Branch and push your changes again or revert the unwanted changes with another commit.

Reply:
@usame-algan, I have removed the unwanted changes.